### PR TITLE
Package rename

### PR DESCRIPTION
**Pull request by CuriousVini**
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 48.920 s
[INFO] Finished at: 2019-04-08T21:01:27-07:00
[INFO] ------------------------------------------------------------------------
```

image

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,18 +55,18 @@
...
 
   <issueManagement>
-    <url>https://issues.cask.co/browse/HYDRATOR</url>
+    <url>https://issues.cask.co/browse/CDAP</url>
   </issueManagement>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
```
Comment (bdmogal) [inline pom.xml:69]: hmmm, in release branch it shouldn't be snapshot, right?

**Review by bdmogal — APPROVED** [review]
1 comment. Fix and merge.

File: pom.xml (hunk 1 above)
Comment (CuriousVini) [reply pom.xml:69]: That is intentional otherwise compilation would fail since we have not released 6.0.0 artifacts of cdap yet.

Comment (bdmogal) [reply pom.xml:69]: is that what we're doing everywhere? If so, merge.

Comment (CuriousVini) [reply pom.xml:69]: that is same everywhere. Also without snapshot version, compilation will fail because 6.0.0 artifacts are not released.

**Pull request merged**